GoedeDoelenHelpen/ClientApp/src/app/app.component.ts

```diff
@@ -12,9 +12,9 @@ export class AppComponent {
   title = 'app';
 
   // icons
-  iconfolder = './assets/images/';
-  constructor(iconReg: MatIconRegistry, sanitizer: DomSanitizer) {
-    iconReg.addSvgIcon('logo', sanitizer.bypassSecurityTrustResourceUrl(this.iconfolder + 'LOGO_NO_TEXT.svg'))
-           .addSvgIcon('logo_text', sanitizer.bypassSecurityTrustResourceUrl(this.iconfolder + 'LOGO_TEXT.svg'));
+  iconfolder = './../assets/images/';
+  constructor(private iconReg: MatIconRegistry, sanitizer: DomSanitizer) {
+    this.iconReg.addSvgIcon('logo', sanitizer.bypassSecurityTrustResourceUrl(this.iconfolder + 'LOGO_NO_TEXT.svg'));
```

Comment:
klinkt niet echt veilig? kan je het niet ergens plaatsen dat het wel een trusted resource is.

Reply:
Helaas niet, reden hiervoor lijkt dat het een registry is van angular material is. Ook maakt het niet echt uit, de Dom Sanatizer is bedoeld om cross site scripting tegen te gaan, wat hier niet echt gevaarlijk is omdat het niet om user input gaat.
Meer info hier: https://netbasal.com/angular-2-security-the-domsanitizer-service-2202c83bd90